webapp/src/ts/modules/reports/reports.component.ts

```diff
@@ -324,20 +324,16 @@ export class ReportsComponent implements OnInit, AfterViewInit, OnDestroy {
     this.scrollLoaderProvider.init(scrollCallback);
   }
 
-  search(force = false) {
+  search() {
```

Comment:
The way out from the reports content page when on mobile is to navigate to the reports list; I change the function to load the list just like [Contacts do](https://github.com/medic/cht-core/blob/47c72bba9ce3efc4115f4ce22cc39c3a5a118e4f/webapp/src/ts/modules/contacts/contacts.component.ts#L408)